Warnings on compilation when not using OpenMP

I added

```c++
// [[Rcpp::depends(RcppProgress)]]
#include <progress.hpp>
#include <progress_bar.hpp>
```
to my code, and now I warnings during compilation:

```c++
C:/---------/R/win-library/3.4/RcppProgress/include/interruptable_progress_monitor.hpp:101:0: warning: ignoring #pragma omp critical [-Wunknown-pragmas]
 #pragma omp critical
 ^
C:/---------/R/win-library/3.4/RcppProgress/include/interruptable_progress_monitor.hpp:143:0: warning: ignoring #pragma omp atomic [-Wunknown-pragmas]
 #pragma omp atomic
 ^
C:/---------/R/win-library/3.4/RcppProgress/include/interruptable_progress_monitor.hpp:149:0: warning: ignoring #pragma omp critical [-Wunknown-pragmas]
 #pragma omp critical
```

--- a/inst/examples/RcppProgressArmadillo/src/example.cpp
+++ b/inst/examples/RcppProgressArmadillo/src/example.cpp
@@ -41,7 +41,9 @@ void test_multithreaded_omp2(int max, int nb, int threads, bool display_progress
 #endif
 
 	Progress p(max, display_progress); // create the progress monitor
+#ifdef _OPENMP
 #pragma omp parallel for schedule(dynamic)
+#endif
 	for (int i = 0; i < max; ++i) {
 		if ( p.increment() ) { // the only way to exit an OpenMP loop
 			your_long_computation(nb);

--- a/inst/examples/RcppProgressExample/src/example.cpp
+++ b/inst/examples/RcppProgressExample/src/example.cpp
@@ -35,7 +35,9 @@ void test_multithreaded_omp2(int max, int nb, int threads, bool display_progress
 #endif
 
 	Progress p(max, display_progress); // create the progress monitor
+#ifdef _OPENMP
 #pragma omp parallel for schedule(dynamic)
+#endif
 	for (int i = 0; i < max; ++i) {
 		if ( p.increment() ) { // the only way to exit an OpenMP loop
 			your_long_computation(nb);

--- a/inst/include/interruptable_progress_monitor.hpp
+++ b/inst/include/interruptable_progress_monitor.hpp
@@ -98,7 +98,9 @@ public: // ===== PBLIC MAIN INTERFACE =====
 	 * request computation abortion
 	 */
 	void abort() {
+#ifdef _OPENMP
 #pragma omp critical
+#endif
 		_abort = true;
 
 	}
@@ -140,13 +142,17 @@ public: // ===== methods for MASTER thread =====
 public: // ===== methods for non-MASTER threads =====
 
 	bool atomic_increment(unsigned long amount=1) {
+#ifdef _OPENMP
 #pragma omp atomic
+#endif
 		_current+=amount;
 		return ! is_aborted();
 	}
 
 	bool atomic_update(unsigned long current) {
+#ifdef _OPENMP
 #pragma omp critical
+#endif
 		_current=current;
 		return ! is_aborted();
 	}

--- a/src/tests.cpp
+++ b/src/tests.cpp
@@ -36,7 +36,9 @@ void test_multithreaded_omp(int max, int nb, int threads, bool display_progress)
 #endif
 
 	Progress p(max, display_progress); // create the progress monitor
+#ifdef _OPENMP
 #pragma omp parallel for schedule(dynamic)
+#endif
 	for (int i = 0; i < max; ++i) {
 		if ( ! p.is_aborted() ) { // the only way to exit an OpenMP loop
 			long_computation(nb);

